ui/components/MeshConfigComponent.js

```diff
@@ -744,54 +722,32 @@ function MesherySettingsNew({ classes, updateProgress, operatorState, k8sconfig
                                     }
-                                    aria-label="meshSync"
-                                  >
-                                    <Chip
-                                      label={'MeshSync'}
-                                      style={meshSyncState === DISABLED ? { opacity: 0.5 } : {}}
-                                      onClick={() => handleMeshSyncClick(rowMetaData.rowIndex)}
-                                      icon={
-                                        <img
-                                          src="/static/img/meshsync.svg"
-                                          className={classes.icon}
-                                        />
-                                      }
-                                      variant="outlined"
-                                      data-cy="chipMeshSync"
-                                    />
-                                  </Tooltip>
+                                    title={'MeshSync'}
+                                    style={meshSyncState === DISABLED ? { opacity: 0.5 } : {}}
+                                    handlePing={() => handleMeshSyncClick(rowMetaData.rowIndex)}
+                                    icon="/static/img/meshsync.svg"
+                                  />
                                 </ListItem>
                               </List>
                             </Grid>
                             <Grid item xs={12} md={4}>
                               <List>
                                 <ListItem>
-                                  <Tooltip
-                                    title={
+                                  <ConnectionChip
+                                    tooltip={
                                       natsState === 'Not Active'
                                         ? 'Not Available'
                                         : `Reconnect NATS`
                                     }
-                                    aria-label="nats"
-                                  >
-                                    <Chip
-                                      label={'NATS'}
-                                      onClick={() => handleNATSClick(rowMetaData.rowIndex)}
-                                      style={natsState === 'Not Active' ? { opacity: 0.5 } : {}}
-                                      icon={
-                                        <img
-                                          src="/static/img/nats-icon-color.svg"
-                                          className={classes.icon}
-                                        />
-                                      }
-                                      variant="outlined"
-                                      data-cy="chipNATS"
-                                    />
-                                  </Tooltip>
+                                    label={'NATS'}
```

Comment:
```suggestion
                                    label={'Meshery Broker'}
```

Reply:
https://github.com/meshery/meshery/pull/9388/files#diff-9514a9ef762e5d87704d59bd30ae5c0a1744322ddcc1bda1a51034bacf4ff690R795
Same thing here.

Reply:
And here as well.
https://github.com/meshery/meshery/pull/9388/files#diff-9514a9ef762e5d87704d59bd30ae5c0a1744322ddcc1bda1a51034bacf4ff690R800